recipe/meta.yaml

```diff
@@ -1826,7 +1826,8 @@ outputs:
 
     test:
       requires:
-        - python >=3.8
+        # FIXME: https://github.com/conda-forge/dagster-feedstock/issues/297
```

Comment:
This is fine, but should also be fixed at the source with a PR @conda-forge/papermill, and likely a PR to `repodata-patches` to `tighten_depends` on `papermill` vs `python`.

Reply:
I'll see if I can tick those off this week...

Reply:
Part 1 - https://github.com/conda-forge/papermill-feedstock/pull/40

Reply:
Sry, was looking a tad deeper: really just needs to be fixed at the source, so it this case ansiwrap:

- https://github.com/conda-forge/ansiwrap-feedstock/pull/3
- https://github.com/conda-forge/conda-forge-repodata-patches-feedstock/pull/577

Reply:
Right, yes - that's better. It will work for anything depending on `ansiwrap` rather than just fixing `papermill`.